plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -1183,7 +1188,7 @@ static void tokudb_backup_run(THD *thd, const char *dest_dir) {
       tokudb_backup_before_stop_capt_fun, thd,
       tokudb_backup_after_stop_capt_fun, &asce);
 
-  if (exclude_string) my_regfree(&exclude_re);
+  if (exclude_copy_extra.re) delete exclude_copy_extra.re;
```

Comment:
delete nullptr is fine

```suggestion
  delete exclude_copy_extra.re;
```